atmos_model.F90

```diff
@@ -533,12 +533,12 @@ subroutine atmos_model_init (Atmos, Time_init, Time, Time_step)
   type (time_type), intent(in) :: Time_init, Time, Time_step
 !--- local variables ---
   integer :: unit, i
-  integer :: mlon, mlat, nlon, nlat, nlev, sec
+  integer :: mlon, mlat, nlon, nlat, nlev, sec, sec_lastfhzerofh
   integer :: ierr, io, logunit
   integer :: tile_num
   integer :: isc, iec, jsc, jec
   real(kind=GFS_kind_phys) :: dt_phys
-  logical              :: p_hydro, hydro
+  logical              :: p_hydro, hydro, tmpflag
```

Comment:
if this variable is going to stay in the code, you may want to choose a more descriptive name?

Reply:
Updated